documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -170,6 +172,20 @@ You can migrate virtual machines to {virt} from the command line.
 
 include::modules/migrating-virtual-machines-cli.adoc[leveloffset=+2]
 include::modules/obtaining-vmware-fingerprint.adoc[leveloffset=+2]
+
+=== Additional authentication methods for migrations with {osp} source providers
+
+{project-short} versions 2.5 and later support the following authentication methods for migrations with {osp} source providers in addition to the standard username/password credential set:
```

Comment:
@RichardHoch 
A slash means 'and/or', which is not the case here. And it's frowned upon by IBM anyway. I would suggest "username and password credential".